manifests/init.pp

```diff
@@ -93,6 +93,14 @@
 #   The ensure attribute on the service
 # @param service_enable
 #   Whether to enable the service (start at boot)
+# @param service_restart_command
+#   Custom command to use when the service will be restarted (notified by
+#   configuration changes). Will be passed directly to the restart parameter of
+#   the contained service resource. This is useful when you want BIND to reload
+#   its configuration instead of restarting the whole process, for example by
+#   setting `service_restart_command` to `/usr/sbin/service bind9 reload` or
+#   `/usr/sbin/rndc reload`. Default 'undef' so the service resource default is
+#   used.
```

Comment:
Another example can be `/usr/bin/systemctl try-reload-or-restart bind9` (but I like the `undef` default, please keep that :))

Reply:
I have added your example to the parameter documentation!